tests/system.rs

```diff
@@ -214,8 +214,11 @@ where
                 fails += 1;
             }
             Ok(report) => reports.push(report),
-            Err(msg) => {
-                print_mismatches(msg);
+            Err(err) => {
+                match err {
```

Comment:
You could use `if let` here rather than `match`.

Reply:
:+1: Thanks for the feedback, and done.